HACKING.md

```diff
@@ -39,7 +39,7 @@ Buck2 itself via `cargo`:
 git clone https://github.com/facebook/buck2.git
 cd buck2/
 nix develop . # add 'rustc' and 'cargo' to $PATH
-cargo build --release --path=app/buck2
+cargo build --release
```

Comment:
This might be closer to what was intended?
```suggestion
cargo build --release --bin=buck2
```

Reply:
Thanks @themarwhal.